plugins/uefi/efi/fwupdate.c

```diff
@@ -576,17 +576,13 @@ efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *systab)
 	for (i = 0; i < n_updates; i++) {
 		fwup_info(L"Adding new capsule");
 		rc = fwup_add_update_capsule(updates[i], &capsules[j], &cbd_data[j], image);
-		if (EFI_ERROR(rc)) {
-			/* ignore a failing UX capsule */
-			if (rc == EFI_UNSUPPORTED &&
-			    CompareGuid(&updates[i]->info->guid, &ux_capsule_guid) == 0) {
-				fwup_debug(L"GOP unsuitable: %r", rc);
-				continue;
-			}
-			fwup_warning(L"Could not build update list: %r", rc);
-			return rc;
-		}
-		j++;
+		if (!EFI_ERROR(rc))
+			j++;
+	}
+
+	if (!j) {
+		dprint(L"fwupdate: Could not build update list: %r\n", rc);
```

Comment:
`fwup_warning()` perhaps?

Reply:
Yes. Sorry I picked his patch verbatim, I should had adapted the code to fwupd.